src/librustc/Cargo.toml

```diff
@@ -13,12 +13,12 @@ arena = { path = "../libarena" }
 fmt_macros = { path = "../libfmt_macros" }
 graphviz = { path = "../libgraphviz" }
 log = "0.3"
+owning_ref = "0.3.3"
```

Comment:
I can't imagine the license of owning_ref is any more of a problem than, say, LLVM or jemalloc being under a different license (jemalloc is even linked into user code, not just rustc) . But of course IANAL and this is a valid question to ask.

FWIW it wouldn't be hard to write something equivalent to `ErasedBoxRef`, especially if it doesn't need a safe general purpose interface. But, needless to say, it would be quite regrettable if we kept duplicating crates.io packages.